### PR TITLE
feat: Use yaml files to create new issue templates

### PR DESCRIPTION
**Pull request by ghost**
Since GitHub has started new yamls to write issue templates with, it's better to create new templates for users to submit issues according to some requirements and info.

ScreenShots for these templates:

### CODE REVIEW

**Comment by ovflowd** [comment]
Oh, that is super cool! Do you have some Docs references from GitHub regarding this feature?

cc @nodejs/nodejs-dev we could also do this on Nodejs.dev

**Review by aymen94 — APPROVED** [review]

**Comment by aymen94** [comment]
LGTM

**Comment by ghost** [comment]
@ovflowd：Of course, you can refer it at:  https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms.

BTW: nodejs/node has applied it actually, it's really useful for us to force people to submit properly :)

**Comment by osk2** [comment]
Should we also create PR template? (There's a PR for this #4334)

**Review by osk2 — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/03-i18n.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: 🔡 Internationalization and translations For Nodejs Website
+description: Hello. Hola. Salut. Ciao. Здравствуйте. こんにちは.
```
Comment (Aissaoui-Ahmed) [inline .github/ISSUE_TEMPLATE/03-i18n.yml]:
```suggestion
description: Hello. Hola. Salut. Ciao. Здравствуйте. こんにちは. مرحباً
```

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Pull request merged**